Random quote generator added

JamBai2812 opened the pull request:
Hard coded an array of quotes from website. 
Generated three distinct random numbers to pick random quotes form the array.
Added quotes to quote cards.

Commit:
Random quote generator added

Hard coded an array of quotes from website. 
Generated three distinct random numbers to pick random quotes form the array.
Added quotes to quote cards.

MichaelWalker commented on src/Pages/InspirationalQuotesPage/InspirationalQuotesPage.tsx:
```diff
@@ -2,23 +2,106 @@
-    return(
+export function InspirationalQuotesPage() {
+
+    const quotes = [
```
Nice.

To make this even nicer we can do 2 things

1) move it outside of the `function` (at the moment we are re-declaring this array every time we re-render, which slightly wasteful (although computers are fast, so we probably get away with it)

2) give it a type! We can make an interface for these quotes, and then the type system can help us out! :)

JamBai2812 replied:
Thanks, Mike. 
Changes made.

MichaelWalker commented on src/Pages/InspirationalQuotesPage/InspirationalQuotesPage.tsx:
```diff
@@ -2,23 +2,106 @@
+                        quote={quotes[quoteThree].quote}
+                        author={quotes[quoteThree].name}
+                    />
+                </li>
```
can simplify this slightly with 
```jsx
{numbers
    .map(number => quotes[number])
    .map(quote => <li><Quotecard quote={quote.quote} author={quote.author}/></li>)
}
```

MichaelWalker commented on src/Pages/InspirationalQuotesPage/InspirationalQuotesPage.tsx:
```diff
@@ -2,23 +2,106 @@
+    var numbers = [];
+    while(numbers.length < 3){
+        var num = Math.floor(Math.random() * (quotes.length));
+        if(numbers.indexOf(num) === -1) numbers.push(num);
```
this works fine, but we can make it a little more readable with
```js
if (!numbers.includes(num)) {
    numbers.push(num);
}
```

Commit:
Made requested changes

Made interface for hardcoded quote objects
Simplified <li> item generator using ].map
moved array outside function to prevent multiple renders

JamBai2812 commented:
changes made.

MichaelWalker commented on src/Pages/InspirationalQuotesPage/InspirationalQuotesPage.tsx:
```diff
@@ -2,23 +2,103 @@
+        }
+    }
+    
+    const quoteOne: number = numbers[0];
+    const quoteTwo: number = numbers[1];
+    const quoteThree: number = numbers[2];
```
We don't need these any more either! :)

MichaelWalker reviewed (approved):
Could delete some unused things, but looks good!

JamBai2812 committed:
Merge branch 'master' into InspirationalQuotesObject
